### DISCUSSION TITLE
Read frequency from SDM72

### DISCUSSION THREAD

**Discussion by ReneHezser** [arendst/Tasmota#16459] [Q&A] [answered]
I've [Added frequency reading to SDM72](https://github.com/ReneHezser/Tasmota/commit/3f6b4a9825bb1fe911a8e1ce6f9ec359a2d2eea8)

What I am struggling with, is the resolution. It is shown as 50 Hz on the website and sent as 50 to MQTT. If I increase the log level to 4, I can see that the responses actually have 49.92 and 49.94. How can this resolution be sent to MQTT and shown on the website?

![image](https://user-images.githubusercontent.com/5988767/188897789-021004dd-904c-4768-91d1-f43e1531ca08.png)

**Comment by arendst** [comment, marked as answer]
Try ``FreqRes 2``

**Reply by ReneHezser** [reply]
Thanks a lot. I stumbled across this in the code, but somehow did not realise it is a command.  ¯\_(ツ)_/¯

**Reply by ReneHezser** [reply]
Once it is connected to three phases (and working), I'll create a pull request.
